### ISSUE TITLE
Static references not working

### ISSUE THREAD

**Issue by hworld** [bmewburn/intelephense#63]
```
<?php

class Task
{
	const STATUS_PENDING = 'pending';

	public static function test() {
		return true;
	}

	public static function run() {
		static::test();
		static::STATUS_PENDING;
	}
}
```

`static::` won't trigger any intellisense. Hover and goto definition doesn't work for anything in the `run` function either. Changing to `self::` does work.

**Comment by bmewburn** [comment]
resolved in v0.8.1

**Comment by hworld** [comment]
Thanks for the fix! This extension is the best. =]
